electron/js/menu/system.js

```diff
@@ -390,11 +390,11 @@ const changeLocale = language => {
 };
 
 module.exports = {
-  createMenu: () => {
+  createMenu: isFullScreen => {
     if (environment.platform.IS_MAC_OS) {
       menuTemplate.unshift(darwinTemplate);
       windowTemplate.submenu.push(separatorTemplate, showWireTemplate, separatorTemplate, toggleFullScreenTemplate);
-      toggleFullScreenTemplate.checked = settings.restore(SETTINGS_TYPE.FULL_SCREEN, false);
+      toggleFullScreenTemplate.checked = isFullScreen || false;
```

Comment:
Weird condition on an or. Let's put this as the default in the param instead as we are overwriting a false with a false. And we are actually doing it twice.